pyproject.toml

```diff
@@ -69,12 +69,6 @@ apm = [
 em = [
     "pynxtools-em",
 ]
-mpes = [
-    "pynxtools-mpes",
-]
-xps = [
-    "pynxtools-xps",
-]
```

Comment:
This should be reenabled before we actually merge this.

Reply:
I kept them out only for testing purposes. They will be repositioned again.

Reply:
They are back to place.